### ISSUE TITLE
Cannot download files with size 0

### ISSUE THREAD

**Issue by evenbrenden** [jotta/jotta-cli-issues#111]
**jotta-cli release (`jotta-cli version`):**
0.6.24251

**Description of problem:**
When attempting to download files using `jotta-cli download`, `jottad` gives up on files with size 0 (e.g. files produced with `touch`). Backup/upload and download via web works fine for the same files. Downloading files with non-zero size from the same source also works fine.

**Expected:**
Successful download of files with size 0.

**jotta-cli status (`jotta-cli status`):**
```
OK
```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```
[PID and timestamp] Giving up download after 4 retries. remotepath: [path to file]
```

**Additional info:**
Downloading to LUKS-encrypted ext4 file system.

**Comment by Kimbsen** [comment]
Thanks for the report! 

I'll make an internal issue and it should get in a future release.

**Comment by Kimbsen** [comment]
We released 0.8.36055 today which should resolve this.
